web3j/src/main/java/com/vmware/ethereum/service/SecureTokenApi.java

```diff
@@ -52,8 +64,13 @@
   private final TokenConfig config;
   private final Web3j web3j;
   private final TransactionManager transactionManager;
+  private final TransactionManager queuedTransactionManager;
+  private final TransactionReceiptProcessor queuedTransactionReceiptProcessor;
   private final String senderAddress;
+  private final Credentials credentials;
   private SecurityToken token;
+  private String contractAddress;
+  private String txHash;
```

Comment:
What is the purpose of having this instance variable?

Reply:
contractAddress and txHash variables are used to store contract address after deployment and transaction hash